native/cocos/platform/android/java/src/com/cocos/lib/CocosWebView.java

```diff
@@ -26,6 +26,7 @@ of this software and associated engine source code (the "Software"), a limited,
 package com.cocos.lib;
 
 import android.annotation.SuppressLint;
+import android.app.Activity;
```

Comment:
Why need to import it?

Reply:
It may be a mistake. I'll try to delete this line.

Reply:
Has removed this line.